replica/database.hh

```diff
@@ -1749,6 +1752,11 @@ public:
     const db::config& get_config() const {
         return _cfg;
     }
+
+    const utils::directories& get_dirs() const {
```

Comment:
database must _not_ be a provider for directories. If any code needs directory, it should get its reference on start, not borrow it via one if its dependent service

Reply:
Sure, fixed :+1: